test/cache/map_cache_test.dart

```diff
@@ -15,8 +15,9 @@
 library quiver.cache.map_cache_test;
 
 import 'dart:async';
-import 'package:test/test.dart';
+
 import 'package:quiver/cache.dart';
+import 'package:test/test.dart';
```

Comment:
nit: we put 3rd-party imports before local imports, so the previous ordering was correct.

Reply:
Done